cpp/order/Steinhardt.cc

```diff
@@ -2,6 +2,7 @@
 // This file is from the freud project, released under the BSD 3-Clause License.
```

Comment:
Since Steinhardt doesn't currently support two sets of points, these changes all seem fine; we'll just need to be extra careful with it when we do update it to enable that.